### PR TITLE
Resolve build warnings on Console UI

### PR DESCRIPTION
**Pull request by anukritijha**
### Type of change

- Refactoring

### Description

Remove Warnings from Console UI

### CODE REVIEW

**Comment by anukritijha** [comment]
@enmasse-ci run tests testcase=FirefoxConsoleTest

File: console/console-init/ui/src/modules/address-space/dialogs/CreateAddressSpace/Configuration.tsx [hunk 1]
```diff
@@ -147,7 +147,7 @@ export const Configuration: React.FunctionComponent<IConfiguration> = ({
                 description:
                   plan.spec.shortDescription || plan.spec.longDescription
               };
-            }
+            } else return {};
```
Comment (ajaypratap003) [inline console/console-init/ui/src/modules/address-space/dialogs/CreateAddressSpace/Configuration.tsx:150]: Not sure, why did you add else return {}, I'm asuming that this is added for remove warning.
It might be cause to add empty object {} in addressSpacePlans in case else condition execute. Can you please verify this case what will happned if else part execute.
